### PR TITLE
Update koKR.lua

### PR DESCRIPTION
**Pull request by WetU**

### CODE REVIEW

File: Nighthold/Locales/koKR.lua [hunk 1]
```diff
@@ -47,7 +47,7 @@ end
 L = BigWigs:NewBossLocale("Star Augur Etraeus", "koKR")
 if L then
 	L.yourSign = "당신의 별자리"
-	L.with = "같은"
+	L.with = ":"
```
Comment (funkydude) [inline Nighthold/Locales/koKR.lua]: This doesn't look right.

File: Nighthold/Locales/koKR.lua [hunk 2]
```diff
@@ -47,7 +47,7 @@ end
 L = BigWigs:NewBossLocale("Star Augur Etraeus", "koKR")
 if L then
 	L.yourSign = "당신의 별자리"
-	L.with = "같은"
+	L.with = "with"
```
Comment (funkydude) [inline Nighthold/Locales/koKR.lua]: If you're not sure what to translate this with, just comment it out `--`

File: Nighthold/Locales/koKR.lua [hunk 3]
```diff
@@ -71,6 +71,8 @@ end
 
 L = BigWigs:NewBossLocale("Gul'dan", "koKR")
 if L then
+	L.warmup_trigger = "부서진 해변에서의 치욕을 잊었느냐?"
```
Comment (funkydude) [inline Nighthold/Locales/koKR.lua]: Is this the whole yell? If it isn't, just add the whole yell next to it commented out `--`

Comment (WetU) [inline Nighthold/Locales/koKR.lua]: Added whole messages

**Pull request merged**